### PR TITLE
fields should be linted against snake case not camel case.

### PR DESCRIPTION
**Pull request by aorban-isp**

### CODE REVIEW

**Review by mbrunner-isp — APPROVED** [review]
lgtm

File: fixtures/disable/openapi.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ paths:
       tags:
         - Test
       parameters:
-        - name: startedUtc
+        - name: started_utcs
```
Comment (mbrunner-isp) [inline fixtures/disable/openapi.yaml:24]: nit: was the extra "s" intententional?

**Review by mbrunner-isp — COMMENTED** [review]
just one nit

File: fixtures/disable/openapi.yaml (hunk 1 above)
Comment (aorban-isp) [reply fixtures/disable/openapi.yaml]: nope fixed

File: fixtures/lint/casing-fail.yaml [hunk 2]
```diff
@@ -21,11 +21,11 @@ paths:
       tags:
         - Test
       parameters:
-        - name: some_query_param
+        - name: someQueryParam
```
Comment (mbordenet) [inline fixtures/lint/casing-fail.yaml:24]: Why is this "going the other direction" from snake to camel?

File: fixtures/lint/casing-properties-fail.yaml [hunk 3]
```diff
@@ -21,13 +21,13 @@ paths:
...
-        '200':
+        "200":
           description: Response description
           content:
             application/json:
               schema:
                 type: object
                 properties:
-                  foo_bar:
+                  fooBar:
```
Comment (mbordenet) [inline fixtures/lint/casing-properties-fail.yaml:31]: Apologies if I'm missing the point, but why wouldn't this be snake case, as well?

Comment (mbrunner-isp) [reply fixtures/lint/casing-properties-fail.yaml:31]: This is a failure test case so making it camel case will make the test (correctly) fail.

File: fixtures/lint/casing-fail.yaml (hunk 2 above)
Comment (mbrunner-isp) [reply fixtures/lint/casing-fail.yaml:24]: Same comment as below - it's a failure test case!

**Pull request merged**